glide.yaml

```diff
@@ -13,6 +13,8 @@ import:
   version: dev
 - package: go.uber.org/thriftrw
   version: dev
+- package: github.com/anmitsu/go-shlex
+- package: github.com/jessevdk/go-flags
```

Comment:
These are needed to install thriftrw and yarpc plugin since the install is looking into vendor for these imports.